daemons/dss-sync-sfn/app.py

```diff
@@ -109,6 +115,9 @@ def launch_from_operator_queue(event, context):
         if exists(dest_replica, key):
             logger.info("Key %s already exists in %s, skipping sync", key, dest_replica)
             continue
+        if key.startswith(BLOB_PREFIX) and not BLOB_KEY_REGEX.match(key):
+            logger.info("Key %s does not match blob key format, skipping sync", key)
```

Comment:
Nit: I notice we are recycling strings a lot. Consider moving logs strings to a central reusable location or variable.

Reply:
@Bento007 DRY is an admirable pursuit, but there is a tradeoff: reduced readability.